skimage/morphology/tests/test_misc.py

```diff
@@ -211,7 +280,7 @@ def test_labeled_image_holes():
         dtype=bool,
     )
     with expected_warnings(['returned as a boolean array']):
-        observed = remove_small_holes(labeled_holes_image, area_threshold=3)
+        observed = remove_small_holes(labeled_holes_image, max_size=3)
```

Comment:
```suggestion
        observed = remove_small_holes(labeled_holes_image, max_size=2)
```
Just to be consistent. But the test passes anyway, since the holes are all smaller than or equal in size to 2 and, hence, to 3 as well.